tools/setup_main.py.in

```diff
@@ -33,6 +33,9 @@ setup(
         "console_scripts": [
              "pybind11-config = pybind11.__main__:main",
         ]
```

Comment:
```suggestion
        ],
```

Reply:
Thanks, sorry, forgot to push the correction till just now.

Reply:
I normally would put this in setup.cfg, but it's only valid for the non-global package, not both. :)